docs/get-started/Velox.md

```diff
@@ -153,31 +153,22 @@ mvn clean package -Pbackends-velox -Prss -Pspark-3.3 -DskipTests
 
 ## 2.3 HDFS support
 
-Hadoop hdfs support is ready via the [libhdfs3](https://github.com/apache/hawq/tree/master/depends/libhdfs3) library. The libhdfs3 provides native API for Hadoop I/O without the drawbacks of JNI. It also provides advanced authentatication like Kerberos based. Please note this library has serveral depedencies which may require extra installations on Driver and Worker node.
-On Ubuntu 20.04 the required depedencis are libiberty-dev, libxml2-dev, libkrb5-dev, libgsasl7-dev, libuuid1, uuid-dev. The packages can be installed via below command:
-```
-sudo apt install -y libiberty-dev libxml2-dev libkrb5-dev libgsasl7-dev libuuid1 uuid-dev
-```
-To build Gluten with HDFS support, below command is provided:
-```
-cd /path_to_gluten/ep/build-velox/src
-./get_velox.sh --enable_hdfs=ON
-./build_velox.sh --enable_hdfs=ON
+Hadoop hdfs support is ready via the [libhdfs3](https://github.com/apache/hawq/tree/master/depends/libhdfs3) library. The libhdfs3 provides native API for Hadoop I/O without the drawbacks of JNI. It also provides advanced authentication like Kerberos based. Please note this library has several dependencies which may require extra installations on Driver and Worker node.
 
-cd /path_to_gluten/cpp
-mkdir build
-cd build
-cmake -DBUILD_VELOX_BACKEND=ON -DENABLE_HDFS=ON ..
-make -j
+### Build with HDFS support
+To build Gluten with HDFS support, below command is suggested:
 
+```
 cd /path_to_gluten
-mvn clean package -Pbackends-velox -Prss -Pspark-3.2 -Pfull-scala-compiler -DskipTests -Dcheckstyle.skip
+./dev/buildbundle-veloxbe.sh --enable_hdfs=ON
 ```
-It is supported to access data on different HDFS endpoints.
-The endpoint info (hdfs://host:port) contained in a given hdfs file path will be used to initialize an hdfs client.
 
-If your HDFS is in HA mode, you need to set the [LIBHDFS3_CONF](https://github.com/apache/hawq/blob/e9d43144f7e947e071bba48871af9da354d177d0/depends/libhdfs3/src/client/Hdfs.cpp#L173) environment variable to specify hdfs client configuration file.
-An alternative, you can also make a [hdfs-client.xml](https://github.com/apache/hawq/blob/e9d43144f7e947e071bba48871af9da354d177d0/depends/libhdfs3/src/client/Hdfs.cpp#L185) and upload it to cluster.
+### Configuration about HDFS support
+ 
+HDFS uris (hdfs://host:port) will be extracted from a valid hdfs file path to initialize hdfs client, you do not need to specify it explicitly.
+
+libhdfs3 need a configuration file and has a little different from hdfs-site.xml and core-site.xml, you can find an example config file from [here](https://github.com/apache/hawq/blob/e9d43144f7e947e071bba48871af9da354d177d0/src/backend/utils/misc/etc/hdfs-client.xml).
```

Comment:
Suggestions:
needs a configuration file
it is a little different from

Reply:
Already updated.